sdk/storage/azblob/service/examples_test.go

```diff
@@ -288,6 +288,8 @@ func Example_service_Client_NewClientWithUserDelegationCredential() {
 	if !ok {
 		panic("AZURE_STORAGE_ACCOUNT_NAME could not be found")
 	}
+	containerName := "testContainer"
```

Comment:
Nit, looks like this should be a `const`

Reply:
Done